Error: Cannot find module './createHandler' from 'index.cjs.js'

**Describe the bug**
I just got the package updated from v0.5.1 to v0.5.2 and I see this error now:

```
Cannot find module './createHandler' from 'index.cjs.js'
```

**To Reproduce**
Steps to reproduce the behavior:
1. Clone [this repo](https://github.com/yurist38/linkedin-video-downloader)
2. Checkout to the branch `issue-jest-chrome`
3. Run tests `npm t`
4. See error

**Expected behavior**
All tests must pass without any errors and coverage 100%.

**Desktop (please complete the following information):**
 - OS: MacOS Catalina 10.15.4
 - Node.js: 12.16.1

**Additional context**
Add any other context about the problem here.

--- a/package.json
+++ b/package.json
@@ -26,22 +26,21 @@
   ],
   "scripts": {
     "build": "run-s build:clean build:pro build:types build:copy",
+    "build:clean": "rm -rf lib types",
     "build:copy": "copyfiles -f src/jest-chrome.d.ts types",
     "build:dev": "rollup -c --environment NODE_ENV:development",
     "build:pro": "rollup -c --environment NODE_ENV:production",
     "build:types": "tsc -p tsconfig.d.json",
-    "build:clean": "rm -rf lib types",
     "build:watch": "npm run build:dev -- -w",
     "release:beta": "git push && npm publish --tag beta",
     "release:full": "git push && npm publish",
     "start": "run-p build:watch",
+    "pretest": "npm run build",
     "test": "jest",
-    "preversion": "run-s test build"
+    "preversion": "npm run test"
   },
   "dependencies": {
-    "@bumble/chrome-rxjs": "^0.10.1",
-    "@types/chrome": "^0.0.113",
-    "rxjs": "^6.5.5"
+    "@types/chrome": "^0.0.114"
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^12.0.0",
@@ -53,29 +52,27 @@
     "@types/fs-extra": "^9.0.1",
     "@types/jest": "^25.2.3",
     "@types/jest-in-case": "^1.0.2",
-    "@types/lodash": "^4.14.152",
-    "@types/node": "^14.0.4",
+    "@types/lodash": "^4.14.153",
+    "@types/node": "^14.0.5",
     "@types/power-assert": "^1.5.3",
     "@types/puppeteer": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^2.34.0",
-    "@typescript-eslint/parser": "^2.34.0",
+    "@typescript-eslint/eslint-plugin": "^3.0.2",
+    "@typescript-eslint/parser": "^3.0.2",
     "chrome-promise": "^3.0.5",
-    "copyfiles": "^2.2.0",
-    "delay": "^4.3.0",
-    "eslint": "^7.0.0",
-    "eslint-plugin-jest": "^23.13.1",
+    "copyfiles": "^2.3.0",
+    "eslint": "^7.1.0",
+    "eslint-plugin-jest": "^23.13.2",
     "fs-extra": "^9.0.0",
     "jest": "^26.0.1",
-    "jest-in-case": "^1.0.2",
     "lodash": "^4.17.15",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.0.5",
-    "rollup": "^2.10.5",
+    "rollup": "^2.11.2",
     "ts-jest": "^26.0.0",
     "tslib": "^2.0.0",
     "typescript": "^3.9.3"
   },
   "peerDependencies": {
-    "jest": "^26.0.1"
+    "jest": "^24.9.0"
   }
 }

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -18,12 +18,6 @@ export default [
         sourcemap: true,
       },
     ],
-    external: [
-      'chrome-promise',
-      '@bumble/chrome-rxjs',
-      'rxjs',
-      'rxjs/operators',
-    ],
     plugins: [
       typescript({
         tsconfig: './tsconfig.base.json',

--- a/tests/demo-cjs.test.js
+++ b/tests/demo-cjs.test.js
@@ -1,0 +1,103 @@
+const { chrome } = require('../lib/index.cjs')
+
+test('chrome api events', () => {
+  const listenerSpy = jest.fn()
+  const sendResponseSpy = jest.fn()
+
+  chrome.runtime.onMessage.addListener(listenerSpy)
+
+  expect(listenerSpy).not.toBeCalled()
+  expect(chrome.runtime.onMessage.hasListeners()).toBe(true)
+
+  chrome.runtime.onMessage.callListeners(
+    { greeting: 'hello' }, // message
+    {}, // MessageSender object
+    sendResponseSpy, // SendResponse function
+  )
+
+  expect(listenerSpy).toBeCalledWith(
+    { greeting: 'hello' },
+    {},
+    sendResponseSpy,
+  )
+  expect(sendResponseSpy).not.toBeCalled()
+})
+
+test('chrome api functions', () => {
+  const manifest = {
+    name: 'my chrome extension',
+    manifest_version: 2,
+    version: '1.0.0',
+  }
+
+  chrome.runtime.getManifest.mockImplementation(
+    () => manifest,
+  )
+
+  expect(chrome.runtime.getManifest()).toEqual(manifest)
+  expect(chrome.runtime.getManifest).toBeCalled()
+})
+
+test('chrome api functions with callback', () => {
+  const message = { greeting: 'hello?' }
+  const response = { greeting: 'here I am' }
+  const callbackSpy = jest.fn()
+
+  chrome.runtime.sendMessage.mockImplementation(
+    (message, callback) => {
+      callback(response)
+    },
+  )
+
+  chrome.runtime.sendMessage(message, callbackSpy)
+
+  expect(chrome.runtime.sendMessage).toBeCalledWith(
+    message,
+    callbackSpy,
+  )
+  expect(callbackSpy).toBeCalledWith(response)
+})
+
+test('chrome api functions with lastError', () => {
+  const message = { greeting: 'hello?' }
+  const response = { greeting: 'here I am' }
+
+  // lastError setup
+  const lastErrorMessage = 'this is an error'
+  const lastErrorGetter = jest.fn(() => lastErrorMessage)
+  const lastError = {
+    get message() {
+      return lastErrorGetter()
+    },
+  }
+
+  // mock implementation
+  chrome.runtime.sendMessage.mockImplementation(
+    (message, callback) => {
+      chrome.runtime.lastError = lastError
+
+      callback(response)
+
+      // lastError is undefined outside of a callback
+      delete chrome.runtime.lastError
+    },
+  )
+
+  // callback implementation
+  const lastErrorSpy = jest.fn()
+  const callbackSpy = jest.fn(() => {
+    if (chrome.runtime.lastError) {
+      lastErrorSpy(chrome.runtime.lastError.message)
+    }
+  })
+
+  // send a message
+  chrome.runtime.sendMessage(message, callbackSpy)
+
+  expect(callbackSpy).toBeCalledWith(response)
+  expect(lastErrorGetter).toBeCalled()
+  expect(lastErrorSpy).toBeCalledWith(lastErrorMessage)
+
+  // lastError has been cleared
+  expect(chrome.runtime.lastError).toBeUndefined()
+})

--- a/tests/demo-esm.test.js
+++ b/tests/demo-esm.test.js
@@ -1,0 +1,101 @@
+import { chrome } from '../lib/index.esm'
+
+test('chrome api events', () => {
+  const listenerSpy = jest.fn()
+  const sendResponseSpy = jest.fn()
+
+  chrome.runtime.onMessage.addListener(listenerSpy)
+
+  expect(listenerSpy).not.toBeCalled()
+  expect(chrome.runtime.onMessage.hasListeners()).toBe(true)
+
+  chrome.runtime.onMessage.callListeners(
+    { greeting: 'hello' }, // message
+    {}, // MessageSender object
+    sendResponseSpy, // SendResponse function
+  )
+
+  expect(listenerSpy).toBeCalledWith(
+    { greeting: 'hello' },
+    {},
+    sendResponseSpy,
+  )
+  expect(sendResponseSpy).not.toBeCalled()
+})
+
+test('chrome api functions', () => {
+  const manifest = {
+    name: 'my chrome extension',
+    manifest_version: 2,
+    version: '1.0.0',
+  }
+
+  chrome.runtime.getManifest.mockImplementation(() => manifest)
+
+  expect(chrome.runtime.getManifest()).toEqual(manifest)
+  expect(chrome.runtime.getManifest).toBeCalled()
+})
+
+test('chrome api functions with callback', () => {
+  const message = { greeting: 'hello?' }
+  const response = { greeting: 'here I am' }
+  const callbackSpy = jest.fn()
+
+  chrome.runtime.sendMessage.mockImplementation(
+    (message, callback) => {
+      callback(response)
+    },
+  )
+
+  chrome.runtime.sendMessage(message, callbackSpy)
+
+  expect(chrome.runtime.sendMessage).toBeCalledWith(
+    message,
+    callbackSpy,
+  )
+  expect(callbackSpy).toBeCalledWith(response)
+})
+
+test('chrome api functions with lastError', () => {
+  const message = { greeting: 'hello?' }
+  const response = { greeting: 'here I am' }
+
+  // lastError setup
+  const lastErrorMessage = 'this is an error'
+  const lastErrorGetter = jest.fn(() => lastErrorMessage)
+  const lastError = {
+    get message() {
+      return lastErrorGetter()
+    },
+  }
+
+  // mock implementation
+  chrome.runtime.sendMessage.mockImplementation(
+    (message, callback) => {
+      chrome.runtime.lastError = lastError
+
+      callback(response)
+
+      // lastError is undefined outside of a callback
+      delete chrome.runtime.lastError
+    },
+  )
+
+  // callback implementation
+  const lastErrorSpy = jest.fn()
+  const callbackSpy = jest.fn(() => {
+    if (chrome.runtime.lastError) {
+      lastErrorSpy(chrome.runtime.lastError.message)
+    }
+  })
+
+  // send a message
+  chrome.runtime.sendMessage(message, callbackSpy)
+
+  expect(callbackSpy).toBeCalledWith(response)
+  expect(lastErrorGetter).toBeCalled()
+  expect(lastErrorSpy).toBeCalledWith(lastErrorMessage)
+
+  // lastError has been cleared
+  expect(chrome.runtime.lastError).toBeUndefined()
+})

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -1,19 +1,19 @@
 {
   "compilerOptions": {
-    "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "lib": ["es2017", "dom", "esnext.array"],
-    "module": "CommonJS",
+    "module": "ES2015",
     "moduleResolution": "node",
     "noImplicitReturns": true,
     "noUnusedLocals": true,
     "resolveJsonModule": true,
+    "skipLibCheck": true,
     "sourceMap": true,
     "strict": true,
     "target": "es2017"
   },
-  "exclude": ["lib", "playground"],
+  "exclude": ["lib", "playground", "tests"],
   "typeAcquisition": {
     "enable": true
   }

